### PR TITLE
api: add DescribeContinuousBackups endpoint to AWS - Dynamodb

### PR DESCRIPTION
**Pull request by angethuy**
### Description

This PR adds a new `DescribeContinuousBackups` endpoint to the AWS dynamodb API and implements it in the dynamodb service and module. This endpoint can be used to retrieve backup information about a given table. 

### Testing Performed

Unit tests for success/error cases

### GitHub Issue

Fixes #

### CODE REVIEW

File: api/aws/dynamodb/v1/dynamodb.proto [hunk 1]
```diff
@@ -182,6 +191,35 @@ message AttributeDefinition {
   string attribute_type = 2;
 }
 
+// Represents the description for a table's continuous backups
```
Comment (scarlettperry) [inline api/aws/dynamodb/v1/dynamodb.proto:194]:
```suggestion
// Represents the description for a table's continuous backups and point in time recovery
```

File: backend/service/aws/dynamodb.go [hunk 2]
```diff
@@ -140,6 +141,15 @@ func newProtoForTable(t *types.TableDescription, account, region string) *dynamo
 	return ret
 }
 
+func newProtoForContinuousBackups(t *types.ContinuousBackupsDescription, account string, region string) *dynamodbv1.ContinuousBackups {
```
Comment (scarlettperry) [inline backend/service/aws/dynamodb.go:144]: from the docs, looks like `ContinuousBackupsStatus` is required to be in the description output. is that true for `PointInTimeRecoveryDescription`, do we need check for nil?

**Review by scarlettperry — COMMENTED** [review]
looks good! just 1 q

File: backend/service/aws/dynamodb.go (hunk 2 above)
Comment (angethuy) [reply backend/service/aws/dynamodb.go:144]: Good catch tyty, I'll add nil checks for `PointInTimeRecoveryDescription` and its fields

**Review by scarlettperry — APPROVED** [review]

**Pull request merged**